dials_data/definitions/insulin_processed.yml

```diff
@@ -0,0 +1,26 @@
+name: Insulin processed
+author: Richard Gildea (2020)
+license: CC-BY 4.0
+description: >
+  dials-data dataset 'insulin' processed using dials with:
+    $ dials.import $(dials.data get -q insulin)/insulin_1_*.img
+    $ dials.find_spots imported.expt nproc=8
+    $ dials.index imported.expt strong.refl space_group=I23
+    $ dials.refine indexed.{expt,refl}
+    $ dials.integrate refined.{expt,refl} nproc=8
+    $ dials.symmetry integrated.{expt,refl}
+    $ dials.scale symmetrized.{expt,refl}
+
+data:
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/imported.expt
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/strong.refl
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/indexed.expt
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/indexed.refl
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/integrated.expt
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/integrated.refl
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/refined.expt
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/refined.refl
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/scaled.expt
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/scaled.refl
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/symmetrized.expt
+  - url: https://github.com/dials/data-files/raw/30eb93776993f77315f88981d9a31d4e572f2f4a/insulin_processed/symmetrized.refl
```

Comment:
I suspect these commit hashes may need to be changed once https://github.com/dials/data-files/pull/17 is merged.